src/modules/SdnDiag.Gateway/public/Get-SdnGatewayConfigurationState.ps1

```diff
@@ -17,6 +17,7 @@ function Get-SdnGatewayConfigurationState {
     $currentErrorActionPreference = $ErrorActionPreference
     $ProgressPreference = 'SilentlyContinue'
     $ErrorActionPreference = 'SilentlyContinue'
+    $FormatEnumerationLimit = -1
```

Comment:
Unused local var. What's the purpose here?

Reply:
Good writeup of this here: https://devblogs.microsoft.com/powershell-community/how-to-use-formatenumerationlimit/

Basically, when you output objects a lot of times it will condense and not show all the objects in the property. That being said, I need to instead do this within the Export-ObjectToFile function directly, since that is where this will translate to.